### PR TITLE
Footer Updates

### PR DESCRIPTION
**Pull request by ZEDz318**
Removed unnecessary divs and classes
Changed height of footer in small screens by changing grid cols from 4 to 2

### CODE REVIEW

**Comment by ZEDz318** [comment]
I closed the last pr by mistake while syncing main
Also, a couple of observations: 
- It is fine to use "footer" in slot (to have footer be sticky), but the problem would be md screens, since both the footer and header are sticky, the content has very little space. I can have it sticky on sm and lg+ screens and static in md but it's not so straight forward and would be messy 
- on sm screens, the content in the body is huge, thus (on the main page) it takes minutes to get to the footer if it's static
- it is not very apparent on larger screens, but when an item is selected, it's description page is initially focused on the footer

**Comment by flicksolutions** [comment]
> - It is fine to use "footer" in slot (to have footer be sticky), but the problem would be md screens, since both the footer and header are sticky, the content has very little space. I can have it sticky on sm and lg+ screens and static in md but it's not so straight forward and would be messy

just a remark: _sticky_ is not the same as _fixed_ here you mean _fixed_. It's fine how you decided here.

> - on sm screens, the content in the body is huge, thus (on the main page) it takes minutes to get to the footer if it's static

yes, this is a feature, not a bug. But I'm afraid if and when we're adding the Vikus-Viewer we might have to change to a paged-layout so we only show 30 items or so and then have a button there to load more.

> - it is not very apparent on larger screens, but when an item is selected, it's description page is initially focused on the footer

yes, this can be considered a bug. The scroll-position is kept the same after navigation. I will fix this later

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -105,5 +107,27 @@
...
+					src={FooterImg1}
+					alt="Ulrich Becher in Basel"
+				/>
+
+				<enhanced:img
+					src={FooterImg2}
+					alt="Ulrich Becher in Basel"
+				/>
+
+				<p class="h5 md:h6 lg:h5 text-black">
```
Comment (flicksolutions) [inline src/routes/+layout.svelte:123]:
```suggestion
				<p class="h5 md:h6 lg:h5 text-black col-span-2 justify-self-start">
```
As [I mentioned](https://github.com/DHBern/becher-frontend/pull/4#discussion_r1470227513) I wanted the text to span two cols.

File: src/routes/+layout.svelte [hunk 2]
```diff
@@ -105,5 +107,27 @@
 	</svelte:fragment>
 	<!-- Page Route Content -->
 	<slot />
-	<svelte:fragment slot="footer">Page Footer</svelte:fragment>
+	<svelte:fragment slot="pageFooter">
+			<div class="grid grid-cols-2 place-items-center md:!grid-cols-4 gap-1 lg:ml-10 lg:mr-10">
```
Comment (flicksolutions) [inline src/routes/+layout.svelte:111]:
```suggestion
			<div class="grid grid-cols-2 place-items-center md:!grid-cols-5 gap-1 lg:ml-10 lg:mr-10">
```
I'd add another col on md to give the description more room...

File: src/routes/+layout.svelte [hunk 3]
```diff
@@ -105,5 +107,27 @@
...
+				<enhanced:img
+					src={FooterImg2}
+					alt="Ulrich Becher in Basel"
+				/>
+
+				<p class="h5 md:h6 lg:h5 text-black">
+					Ein Prototyp des Deutschen Exilarchivs (DEA) und des Schweizerischen Literarurarchivs (SLA) 
+				</p>
+
+				<p class="h5 text-black">
```
Comment (flicksolutions) [inline src/routes/+layout.svelte:127]:
```suggestion
				<p class="h5 text-black col-span-2 md:col-span-1 justify-self-start md:justify-self-end">
```

**Review by flicksolutions — APPROVED** [review]
ok! I'll add the small changes myself. noted them here just for you to see.

File: src/routes/+layout.svelte (hunk 3 above)
Comment (ZEDz318) [reply src/routes/+layout.svelte:127]: ohh now i see. I miss understood. I thought you meant span as in the verb

**Pull request merged**